WalletWasabi.Fluent/Styles/Button.axaml

```diff
@@ -56,10 +56,47 @@
   <Style Selector="Button.function:disabled PathIcon">
     <Setter Property="Foreground" Value="{DynamicResource ButtonForegroundDisabled}" />
   </Style>
+  <Style Selector="Button.function TextBlock">
+    <Setter Property="VerticalAlignment" Value="Center" />
+  </Style>
   <Style Selector="Button.function:disabled /template/ ContentPresenter#PART_ContentPresenter">
     <Setter Property="Background" Value="Transparent" />
   </Style>
 
+  <Style Selector="Button.function Ellipse.pulsateRect">
+    <Setter Property="VerticalAlignment" Value="Center" />
+    <Setter Property="HorizontalAlignment" Value="Center" />
+    <Setter Property="Width" Value="5" />
+    <Setter Property="Height" Value="5" />
+    <Setter Property="RenderTransformOrigin" Value="50%,50%" />
+  </Style>
+
+  <Style Selector="Button.function.pulsate.active Ellipse.pulsateRect">
+    <Setter Property="Fill" Value="{StaticResource SystemAccentColorLight3}" />
+    <Style.Animations>
+      <Animation IterationCount="Infinite" Duration="0:0:2.3" Easing="{StaticResource FluentEasing}">
+        <KeyFrame Cue="0%">
+          <Setter Property="Opacity" Value="0" />
+          <Setter Property="(ScaleTransform.ScaleX)" Value="0" />
+          <Setter Property="(ScaleTransform.ScaleY)" Value="0" />
+        </KeyFrame>
+        <KeyFrame Cue="60%">
+          <Setter Property="Opacity" Value="1" />
+        </KeyFrame>
+        <KeyFrame Cue="90%">
+          <Setter Property="Opacity" Value="1" />
+          <Setter Property="(ScaleTransform.ScaleX)" Value="40" />
+          <Setter Property="(ScaleTransform.ScaleY)" Value="40" />
+        </KeyFrame>
+        <KeyFrame Cue="100%">
+          <Setter Property="Opacity" Value="0" />
+          <Setter Property="(ScaleTransform.ScaleX)" Value="40" />
+          <Setter Property="(ScaleTransform.ScaleY)" Value="40" />
+        </KeyFrame>
+      </Animation>
+    </Style.Animations>
+  </Style>
```

Comment:
I am not sure if the `pulsateRect` related stuff is at the correct place. This makes me think the pulsing stuff is usable everywhere, but in fact, it would be only usable in the WalletView.

I might be wrong, can we discuss this?

Reply:
yeah that's correct, i moved it to the walletview xaml